Makefile

```diff
@@ -1,3 +1,11 @@
+bdt:
+	# Please set env vars BDT_CLIENT_ID and BDT_CLIENT_SECRET before running
+	docker build -t bdt -f Dockerfiles/Dockerfile.bdt .
+	docker run --rm \
+	-e CLIENT_ID='${BDT_CLIENT_ID}' \
```

Comment:
How do we point this at a specific environment?  Can we have an env var to point it to a "system under test" so we can dynamically point it at `dev.bcda.cms.gov`, `test.bcda.cms.gov`, etc?